.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=827846
```

Comment:
github syntax highlighting doesn't like these comments, but I think they are also valid here.  (Trailing commas also appear to be valid?? :eyes:)

Do you think it's worth adding this line to the `.gitattributes` file?
```
.vscode/*.json jsonc
```

Reply:
We can certainly try and find out what happens!
I just copied/pasted it from other people; it seems like most everyone has these comments in (despite them being so ugly in GitHub)

Reply:
I think GitHub still hates it :(

Reply:
GitHub likes this now :)